### DISCUSSION TITLE
Live Media over TQUIC

### DISCUSSION THREAD

**Discussion by Vaibtan** [Tencent/tquic#342] [Q&A (问题和回答)]
Greetings folks, I am working on a project to transfer live media (videos) over TQUIC (between 2 different machines). I am not well versed with networks and my working knowledge of TQUIC is also limited.  Are there any resources regarding the same or any ideas as to how should I approach this problem?

**Comment by iyangsj** [comment]
You can try integrating TQUIC into open-source live streaming server and client projects. To do this, you need to be familiar with these projects. Additionally, we plan to release integration solutions for some ecosystem projects.
